template/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,34 +1,37 @@
 object Versions {
-    const val BUILD_GRADLE_VERSION = "4.2.1"
+    const val BUILD_GRADLE_VERSION = "7.3.0"
 
-    const val ANDROID_COMPILE_SDK_VERSION = 30
+    const val ANDROID_COMPILE_SDK_VERSION = 33
     const val ANDROID_MIN_SDK_VERSION = 23
-    const val ANDROID_TARGET_SDK_VERSION = 30
+    const val ANDROID_TARGET_SDK_VERSION = 33
 
     const val ANDROID_VERSION_CODE = 1
     const val ANDROID_VERSION_NAME = "1.0.0"
 
     // Dependencies (Alphabet sorted)
     const val ANDROID_COMMON_KTX_VERSION = "0.1.1"
     const val ANDROID_CRYPTO_VERSION = "1.0.0"
-    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.2.1"
-    const val ANDROIDX_CORE_KTX_VERSION = "1.3.0"
-    const val ANDROIDX_FRAGMENT_VERSION = "1.3.3"
-    const val ANDROIDX_LIFECYCLE_VERSION = "2.4.0-alpha02"
+    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.6.0"
+    const val ANDROIDX_CORE_KTX_VERSION = "1.9.0"
+    const val ANDROIDX_CORE_TEST_VERSION = "1.4.0"
```

Comment:
Because the library is named `androidx.test:core:x.x.x`
```suggestion
    const val ANDROIDX_TEST_CORE_VERSION = "1.4.0"
```